lib/IR/IRGen.cpp

```diff
@@ -269,7 +269,8 @@ void IRGenVisitor::post(Node *parent, Node *N) {
 
     auto *dest = builder_.createAllocActivationInst(CC->getName(),
                                                     CC->getResult().getType());
-    builder_.createSplatInst(CC->getName(), dest, 0);
+    // Mark the buffer as initialized.
```

Comment:
nit: Perhaps add a little more info, e.g. "This is safe because the insert insts below which implement the concat must cover the entire buffer" or something?

Reply:
sure, I'll add some more colour.